.github/workflows/examples.yml

```diff
@@ -51,6 +54,12 @@ jobs:
             python $file --pruning > /dev/null
           fi
         done
+
+        for file in `find examples -name '*.ipynb'`
```

Comment:
Should I split this test case like `Run multi-node examples`?

Reply:
I prefer to split.